openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -197,7 +198,12 @@ def transform_level_of_education(self, user_profile, value):  # pylint: disable=
         """ Converts empty string to None, to indicate not set. Replaced by to_representation in version 3. """
         return AccountLegacyProfileSerializer.convert_empty_to_None(value)
 
-    def transform_bio(self, user_profile, value):  # pylint: disable=unused-argument
+    # pylint: disable=unused-argument
```

Comment:
Disable inline so you only disable for the one line. Doing this disables it in the file from here to the bottom.